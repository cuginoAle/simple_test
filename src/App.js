import React from 'react';
import api from 'APIs';
import styled from 'styled-components';
import bg from 'assets/background.png';
import VideoList from './components/containers/VideoList';

const Root = styled.div`
  background-image: url(${bg});
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
`;
function App() {
  return (
    <Root id="root">
      <VideoList getVideos={api.getVideos} />
    </Root>
  );
}

export default App;
