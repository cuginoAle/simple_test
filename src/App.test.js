import { render, waitFor } from '@testing-library/react';
import React from 'react';
import App from 'App';

describe('App', () => {
  it('Should match spanshot', () => {
    const app = waitFor(() => render(<App />));

    expect(app).toMatchSnapshot();
  });
});
