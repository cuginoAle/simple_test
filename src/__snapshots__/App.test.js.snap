// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should match spanshot 1`] = `Promise {}`;
